Source/MDFastBinding/Private/BindingDestinations/MDFastBindingDestination_Property.cpp

```diff
@@ -96,14 +112,55 @@ void UMDFastBindingDestination_Property::SetupBindingItems()
 	Super::SetupBindingItems();
 
 	EnsureBindingItemExists(MDFastBindingDestination_Property_Private::PathRootName
-		, nullptr
+		, GetClass()->FindPropertyByName(GET_MEMBER_NAME_CHECKED(UMDFastBindingDestination_Property, ObjectProperty))
```

Comment:
So does it still let you connect struct values? Just has a "UObject" blue pin color?

Reply:
I probably should have tested this more. It looks like struct values supplied to property destination nodes just don't work at all, since they get static_casted to UObject from `UMDFastBindingDestination_Property::GetPropertyOwner`. I'll write a separate issue for this one.

This change also looks like it fails validation in `UMDFastBindingObject::IsDataValid` for **BindingItemTypeMismatchError** 😞. I think despite that it should work at runtime (since `FMDFastBindingItem::GetValue` will have a value to work with, but I'll only be able to confirm that after fixing the previous issue.

Reply:
We could either put this PR on hold, or revert this specific commit and address those other issues separately.

Reply:
Ooof, yeah there's safety issues with `UMDFastBindingDestination_Property::GetPropertyOwner` and `UMDFastBindingValue_Property::GetPropertyOwner`, they should both be changed to return `void*` and then we could add `GetPropertyOwnerUObject` variants that could be used where necessary.